wagtail/wagtailadmin/views/home.py

```diff
@@ -10,6 +12,11 @@
 from wagtail.wagtailcore.models import Page, PageRevision, UserPagePermissionsProxy
 
 
+User = get_user_model()
+pk_field_name = User._meta.pk.name
+pk_field = User._meta.get_field(pk_field_name)
```

Comment:
Do you need to go through `pk_field_name` here, or will `pk_field = User._meta.pk` do the job?

(Also, since the variable only gets used a bit further down the script, we should give it a more informative name, like `user_pk_field`)